crowbar_framework/app/models/api/node.rb

```diff
@@ -131,7 +140,10 @@ def upgrade
         return false
       end
 
-      reboot_and_wait
+      unless reboot_and_wait
+        save_error_state("Error while rebooting the node")
+        return false
```

Comment:
True. This one is waiting for merge of #935 anyway (so now we can add #960 to the list)